templates/receipt/README.md

```diff
@@ -0,0 +1,31 @@
+Sample receipt document
+======================
+The creator package is used to generate pdf pages using components and simple interfaces. It also provides templates to easily use page components and manipulate their layout with fewer code and generate pdf documents fast. The templates use XML based markup to represent page components and they are converted to creator components at runtime.
```

Comment:
Please follow the pattern the other examples use for the this (e.g. [airplane-ticket](https://github.com/unidoc/unipdf-examples/tree/master/templates/airplane-ticket)). There's no need for all this text in the `README.md` file.

Reply:
yeah, t was in its own directory first. 
Now that it has moved to `templates` directory a simple one liner and a preview image may be enough.